src/arcaflow_plugin_sdk/schema.py

```diff
@@ -4887,13 +4893,14 @@ class ObjectType(ObjectSchema, AbstractType, Generic[ObjectT]):
     You can now use the object_type to unserialize, validate, and serialize properties.
     """  # noqa: E501
 
-    _cls: Type[ObjectT]
-    properties: Dict[str, PropertyType]
+    _cls: Type[ObjectT] = None
+    properties: Dict[str, PropertyType] = None
+    id_unenforced: Optional[bool] = None
```

Comment:
I'm still not quite sure how I feel about this. It's consistent with normal `@dataclass` behavior, but since you're overriding `__init__` the default values will *never* be accessed (without an explicit `ObjectSchema._cls`/etc) and therefore *having* the values here is potentially misleading. (That is, you can't actually change the default by editing these lines, you need to edit the `__init__` implementation instead.)

I'll approve, with some reluctance, just for the misleading documentation here; I'd prefer that you remove the values, as they were before.

Reply:
The problem is the ordering of default values. I added the field id_unenforced, which is an optional field. Now that there is a default value in `ObjectSchema`, `ObjectType` needs to have all of its fields be default. This is just due to the ordering of fields, and the requirement that default values must come first.
Do you have any other proposals for this?

Reply:
My main concern doing that is it will end up with false being present in exported schemas. Whereas if it stays None, I'd imagine that it doesn't. I will need to validate that.

Reply:
In the code as reviewed, this is a schema for a test plugin:
```
{'steps': {'incorrect-return': {'id': 'incorrect-return', 'input': {'objects': {'EmptyTestInput': {'id': 'EmptyTestInput', 'properties': {}, 'id_unenforced': False}}, 'root': 'EmptyTestInput'}, 'outputs': {'success': {'schema': {'objects': {'EmptyTestOutput': {'id': 'EmptyTestOutput', 'properties': {}, 'id_unenforced': False}}, 'root': 'EmptyTestOutput'}, 'error': False}}, 'display': {'name': 'Incorrect Return', 'description': 'A step that returns a bad output which omits the output ID.'}}}}
```

Switching the value passed into the super constructor to `self.id_unenforced` removes the id_unenforced field:
```
{'steps': {'incorrect-return': {'id': 'incorrect-return', 'input': {'objects': {'EmptyTestInput': {'id': 'EmptyTestInput', 'properties': {}}}, 'root': 'EmptyTestInput'}, 'outputs': {'success': {'schema': {'objects': {'EmptyTestOutput': {'id': 'EmptyTestOutput', 'properties': {}}}, 'root': 'EmptyTestOutput'}, 'error': False}}, 'display': {'name': 'Incorrect Return', 'description': 'A step that returns a bad output which omits the output ID.'}}}}
```

And to confirm that it's working as intended as an instance variable specifically I changed the default to True, and it showed up in the exported schema.